playwright/src/utilities/user.ts

```diff
@@ -62,12 +62,17 @@ async function accountsUserSignup(page: Page, url = '', student: Student = new S
   await page.fill('[placeholder="Last name"]', student.last)
   await page.fill('[placeholder="me@myemail.com"]', student.email)
   await page.fill('[placeholder="Password"]', student.password)
-  await page.check('#signup_terms_accepted')
-  await page.click('text=Continue')
```

Comment:
@gregfitch the mobile tests (for iPhone 12) are not picking these existing locators. They dont fail for invisible/inactive locators; looks as if they just skip those lines and moves to the next and when they are not able to find the locator in the new page the test times out. If I change the TOS checkbox & Continue buttons to `page.evaluate` then it works. Now its doing the same for `Finish` button.
Have you encountered this issue before?

Reply:
(Assuming you mean webkit/Safari) I haven't, but using `evaluate` there (_not really what it's meant for_) means that it waits for whatever Promise it is given as an argument. If I had to guess, I would say the webkit version is acting before the DOM element is 'ready' for the action or command (ie a race condition - it sees the element so it sends the action, but the action isn't actually completed). Using evaluate is probably giving the site extra time to get its ducks in a row before sending the action (kind of like making a script sleep for a moment). This ***shouldn't*** happen, but if it is...webkit has always been the difficult child in automated testing.

Reply:
I am thinking to proceed with the `evaluate` and if I still see frequent webkit failures in the CI checks, disable webkit from CI.